tests/integration/test_market.py

```diff
@@ -81,7 +83,7 @@ def test_match_order(bootstrapped_market: Market, stubbed_payer: Account, http_c
 
 
 @pytest.mark.integration
-def test_new_order(
+def test_order_placement_cancellation_cycle(
```

Comment:
Can you make this a sperate test and keep `test_new_order`?

Reply:
If I add a new tests then the new test, say `test_cancel_order` will depend on `test_new_order`. I don't think introducing this kind of dependency is a good idea. Also we will rely on the relative execution order of the two tests, since we want `test_new_order` to be run before `test_cancel_order`.